### PR TITLE
Weolcome pages, context done

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div
+      className="App"
+      // style={{
+      //   backgroundImage: `url("https://source.unsplash.com/1920x1080/?wallpaper")`,
+      // }}
```
Comment (anjali1102) [inline src/App.js:22]: you can remove these

File: src/Pages/WelcomePage/WelcomePage.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.welcome__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  height: 100vh;
+  /* width: 20%; */
+}
```
Comment (anjali1102) [inline src/Pages/WelcomePage/WelcomePage.css:9]: this also

File: src/Pages/WelcomePage/WelcomePage.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+const WelcomePage = () => {
+  const { setUserName } = useExtension();
+
+  function keyDownHandler(e) {
+    if (e.code === "Enter" && e.target.value !== "") {
+      const inputValue =
+        e.target.value.charAt(0).toUpperCase() + e.target.value.slice(1);
+      console.log(inputValue);
+      setUserName(inputValue);
```
Comment (anjali1102) [inline src/Pages/WelcomePage/WelcomePage.js:13]: you can remove this console.log

**Review by anjali1102 — APPROVED** [review]
deploy too as well

File: src/App.js (hunk 1 above)
Comment (ashishkatiyar2404) [reply src/App.js:22]: ok

**Review by MaruthiPothuganti — APPROVED** [review]
PR looks good to merge

**Pull request merged**